packages/prisma/schema.prisma

```diff
@@ -169,6 +171,7 @@ model Recipient {
   email         String        @db.VarChar(255)
   name          String        @default("") @db.VarChar(255)
   token         String
+  templateToken String?
```

Comment:
Why did we introduce this?

Reply:
a unique token (templateToken) is assigned to each recipient in a template. This token is specific to that particular template. When a new document is created from the template, this same token is used to create recipients for the new document. This ensures that the recipients of the document are directly linked to the original template recipients.

Reply:
I tossed it since we can use email instead due to it having a composite unique constraint 😄